freeipa-client/src/main/java/com/sequenceiq/freeipa/client/RetryableFreeIpaClientException.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.freeipa.client;
+
+import java.util.OptionalInt;
+
+public class RetryableFreeIpaClientException extends FreeIpaClientException {
+
+    public static final int MAX_RETRIES = 5;
+
+    public static final int BACKOFF_DELAY = 5000;
+
+    public static final double BACKOFF_MULTIPLIER = 1.5;
```

Comment:
These should probably be configurable with @Value.

Reply:
These are used for annotation configuration in @Retryable. @Retryable requires the fields to be static which can't be configured using @Value. The @Retryable also doesn't support configuration directly via @Value. See https://stackoverflow.com/questions/38088449/how-can-i-make-spring-retryable-configurable for why this can't be done.

Reply:
Done